### DISCUSSION TITLE
iOS18 Issue

### DISCUSSION THREAD

**Discussion by Frogy76** [kishikawakatsumi/WebTranslator#270] [Q&A]
There seems to be a problem with the iOS 18 version. You can log in and the login works, but as soon as you try to translate a text, for example in German, you get an error message that the text cannot be translated. When you leave the dialogue box, the website is translated to Bulgarian. I.e. the translation still seems to work but there seems to be a problem with the language
![image](https://github.com/user-attachments/assets/ca9d462f-3ab3-4992-ac12-01b93c905ae7)

![image](https://github.com/user-attachments/assets/c45d1b19-c6c0-4dd3-af60-aeea9a45fe94)

**Comment by mauz001** [comment]
I have the same problem. No matter what language I choose - Japanese, English, or Korean - it always ends up being translated into Bulgarian

**Comment by kishikawakatsumi** [comment]
The problem is reproduced on my end. I just haven't figured out how to fix it yet.

**Comment by kishikawakatsumi** [comment]
https://github.com/kishikawakatsumi/WebTranslator/pull/288
